InterfacePython/testCommon.py

```diff
@@ -86,20 +84,22 @@ def compareGrids(self, gridA, gridB, bTestRuleNames = True):
         pB = gridB.getQuadratureWeights()
         np.testing.assert_equal(pA, pB, "Quadrature not equal", True)
 
-        pA = gridA.getInterpolationWeights(mX1)
-        pB = gridB.getInterpolationWeights(mX1)
-        np.testing.assert_equal(pA, pB, "Interpolation test 1 not equal", True)
+        # evaluate (values have been loaded)
+        if (gridA.getNumLoaded() > 0):
```

Comment:
I made a mistake here. I didn't realize that the `mX` points are used even without loaded points.

In the current set of tests, the `getInterpolationWeights()` methods test the construction of the Lagrange polynomials which doesn't require `getNumLoaded() > 0`. We should test the Lagrange polynomials even in the case when using CustomTabulated rules.

Defining a set of test points in the 1D case is the correct thing to do here. Your first attempt was the right one.

Reply:
Gotcha. I changed the code back to add the 1D nodes instead of moving around  `aBatchPoints` and changing the location of several if-blocks.